src/index.js

```diff
@@ -146,16 +146,18 @@ if ( !! window.wcCalypsoBridge.isEcommercePlanTrial ) {
 		render: WoocommercePaymentsTaskPage,
 	} );
 
-	if ( window.wcCalypsoBridge.wooExpressIntroductoryOffer ) {
-		registerPlugin( 'wc-calypso-bridge-homescreen-slotfill-banner', {
-			render: CalypsoBridgeIntroductoryOfferBanner,
-			scope: 'woocommerce-admin',
-		} );
-	} else {
-		registerPlugin( 'wc-calypso-bridge-homescreen-slotfill-banner', {
-			render: CalypsoBridgeHomescreenBanner,
-			scope: 'woocommerce-admin',
-		} );
+	if ( window.location.search === '?page=wc-admin' ) {
```

Comment:
@moon0326 I think this is not reliable since there might be other URL parameters used.

I'd suggest to use `getPath` from `@woocommerce/navigation` where it's more reliable

Reply:
@ilyasfoo getPath() returns `location.pathname` and it's hard to determine Woo Home with the returned value AFAIK. Do you see situations where this code won't work?

Reply:
FYI, I've also tried using [determine_current_page](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Admin/PageController.php#L119) and [get_current_page](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Admin/PageController.php#L219) from PageController class. Both report `homescreen` when you're on a task page.

Reply:
Thanks, @moon0326! Unfortunately, I found another problem with the way we conditionally load the SlotFill 😅 

It seems if we navigate to another page first, such as `/wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Foverview` and then navigate to homescreen using the menu bar, no banner will be loaded.

Sorry for going round and round on this, but let's not check for homescreen in the `registerPlugin` layer, but instead in the component before firing the tracks.

Reply:
> Sorry for going round and round on this, but let's not check for homescreen in the registerPlugin layer, but instead in the component before firing the tracks.

No worries. Thank you for catching these cases 🤦 

I'll run a thorough test again.

Reply:
@ilyasfoo Fixed in https://github.com/Automattic/wc-calypso-bridge/pull/1398/commits/6bb67a11658027fa0989e3579f604333be43f61a 🙏